attributes/default.rb

```diff
@@ -193,18 +167,6 @@
 # If you're already using version >= 5.12.0 of the Agent, leave this to false.
 default['datadog']['windows_agent_use_exe'] = false
 
-# Values that differ on Windows
-# The location of the config folder (containing conf.d)
-# The name of the dd agent service
-# The log file directory (see logging section below)
-if node['platform_family'] == 'windows'
-  default['datadog']['config_dir'] = "#{ENV['ProgramData']}/Datadog"
-  default['datadog']['agent_name'] = 'DatadogAgent'
-else
-  default['datadog']['config_dir'] = '/etc/dd-agent'
-  default['datadog']['agent_name'] = 'datadog-agent'
```

Comment:
Ideally, for attributes that aren't defined and shouldn't be used anymore in 4.x, you could add logic to print warnings if they're set.

Note that `config_dir` is now configurable on the Windows Agent since v6.11 (with `APPLICATIONDATADIRECTORY`, see https://docs.datadoghq.com/agent/basic_agent_usage/windows/?tab=agentv6). Doesn't mean that you should add support for this now (3.x didn't support the option OOTB), but, just so you know, we may want to re-introduce this attribute at some point :)